### PR TITLE
Add keyboard shortcut l in breakpoint list to show the location

### PR DESCRIPTION
**Pull request by asmeurer**
c.f. #92

### CODE REVIEW

**Comment by asmeurer** [comment]
Not sure if it should be `L` instead, to match the "goto line" shortcut. Personally I always expect `Enter` to do this, but we probably shouldn't change what that already does.

File: pudb/debugger.py [hunk 1]
```diff
@@ -1109,6 +1110,17 @@ def examine_breakpoint(w, size, key):
                 else:
                     self.update_breakpoints()
 
+        def breakpoint_location(w, size, key):
```
Comment (inducer) [inline pudb/debugger.py]: I think this is largely identical to `show_breakpoint`, bound to `Enter` in that context.

Comment (asmeurer) [reply pudb/debugger.py]: Hmm, am I crazy? I was pressing Enter before, but it opened the dialog. I just tested it now and it does what I want. We definitely don't need this PR if Enter already does this.

Comment (asmeurer) [reply pudb/debugger.py]: Oh this was changed recently and I was using the released version. Sorry for the noise. This can be closed.

**Pull request closed without merging**